tests/target/trailing_comments/hard_tabs.rs

```diff
@@ -12,7 +12,8 @@ impl Foo {
 }
 
 fn lorem_ipsum() {
-	let f = bar(); // Donec consequat mi. Quisque vitae dolor. Integer lobortis. Maecenas id nulla. Lorem.
+	let f = bar(); // Donec consequat mi. Quisque vitae dolor. Integer lobortis. Maecenas id nulla.
+			   // Lorem.
```

Comment:
the alignment seems off.

Reply:
Yeah, I think we need to convert `\t` to spaces when calculating the offset.

Reply:
Filed https://github.com/rust-lang/rustfmt/issues/3950.